src/compile/mark/mark.ts

```diff
@@ -38,7 +44,15 @@ const markCompiler: Record<Mark, MarkCompiler> = {
   trail
 };
 
-export function parseMarkGroups(model: UnitModel): any[] {
+export interface LabelMark extends TextMark {
+  transform: [LabelTransform];
```

Comment:
Do we need an array here? Is there ever more than one LabelTransform?

Reply:
This is a type for an output mark in Vega. If I'm not mistaken, a transform can only be an array?